### PR TITLE
Issue 30 - run transformation

### PR DESCRIPTION
**Pull request by murphyke**
This is not ready yet; still WIP.

### CODE REVIEW

File: pedsnetdcc/primary_keys.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def _check_stmt_err(stmt, force):
     already_exists = (
         hasattr(stmt.err, 'pgcode')
         and stmt.err.pgcode
-        and psycopg2_errorcodes.lookup(
+        and psycopg2_errorcodes.c(
```
Comment (aaron0browne) [inline pedsnetdcc/primary_keys.py]: I have `.lookup` in my code, too. Is it wrong?

File: pedsnetdcc/transform_runner.py [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        join_obj = table
+
+        for transform in TRANSFORMS:
+
+            select_obj, join_obj = transform.modify_select(
+                metadata,
+                table_name,
+                select_obj,
+                join_obj)
+
```
Comment (aaron0browne) [inline pedsnetdcc/transform_runner.py]: Do you need to also call `modify_metadata` here?

**Comment by murphyke** [comment]
The [age transform bug](https://github.com/PEDSnet/pedsnetdcc/pull/56) was the primary obstacle, but since I hadn't finished the tests yet, there are other issues, which I am working on now.

**Comment by murphyke** [comment]
TODO:
* test with id-mapping transformation merged
* add undo function

**Comment by murphyke** [comment]
Ready for review.

Parallel execution of transformations restored. I'm not sure why this was hanging before, although the statements being executed were invalid. I created an extra db test with invalid statements for the heck of it, which passes.

File: pedsnetdcc/transform_runner.py [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+        final_select_obj = select_obj.select_from(join_obj)
+
+        table_sql = str(
+            final_select_obj.compile(
+                dialect=sqlalchemy.dialects.postgresql.dialect()))
+
+        final_sql = 'CREATE UNLOGGED TABLE {0}.{1} AS {2}'.format(
```
Comment (murphyke) [inline pedsnetdcc/transform_runner.py]: From reviewing the merger module, I noticed the lack of `IF EXISTS` here, which would prevent directly restarting an aborted transformation if some or all tables had been created and populated.

**Comment by aaron0browne** [comment]
Very cursory code review for obvious errors did not find anything. Since the tests are passing and we're on a deadline, I think we should merge and move on with a commitment to come back and refactor for maintainability. Agreed?

**Comment by murphyke** [comment]
Yes, agreed.

**Pull request merged**